backend/layers/thirdparty/schema_validator_provider.py

```diff
@@ -65,3 +77,16 @@ def add_labels(self, input_file: str, output_file: str) -> None:
 
     def count_matrix_nonzero(self, matrix):
         return validate.Validator.count_matrix_nonzero(matrix)
+
+    def validate_atac(self, fragment_file, anndata_file) -> Optional[List[str]]:
+        """
+        Validates an ATAC fragment file against an anndata file.
+        """
+        import cellxgene_schema.atac_seq as atac_seq
+
+        return atac_seq.validate_fragment(fragment_file, anndata_file)
```

Comment:
This is installing from `main` right?

I do not see a function called `validate_fragment` in this module: https://github.com/chanzuckerberg/single-cell-curation/blob/main/cellxgene_schema_cli/cellxgene_schema/atac_seq.py

Not totally why coverage isn't being complained about here.

Reply:
whoops, should be `process_fragment`